rpc/src/module/pool.rs

```diff
@@ -241,17 +331,22 @@ impl PoolRpc for PoolRpcImpl {
         let tx: core::TransactionView = tx.into_view();
 
         if let Err(e) = match outputs_validator {
-            Some(OutputsValidator::Default) => {
-                DefaultOutputsValidator::new(self.shared.consensus()).validate(&tx)
+            Some(OutputsValidator::WellKnownScriptsOnly) | None => {
+                WellKnownScriptsOnlyValidator::new(
+                    self.shared.consensus(),
+                    &self.well_known_lock_scripts,
+                    &self.well_known_type_scripts,
+                )
+                .validate(&tx)
             }
-            Some(OutputsValidator::Passthrough) | None => Ok(()),
+            Some(OutputsValidator::Passthrough) => Ok(()),
         } {
             return Err(RPCError::custom_with_data(
                 RPCError::PoolRejectedTransactionByOutputsValidator,
                 format!(
                     "The transction is rejected by OutputsValidator set in params[1]: {}. \
-                    Please set it to passthrough if you really want to send transactions with advanced scripts.",
-                    outputs_validator.unwrap_or(OutputsValidator::Default).json_display()
+                    Please check the related information in https://github.com/nervosnetwork/ckb/wiki/Transaction-%C2%BB-Default-Outputs-Validator",
```

Comment:
* Wiki should add description of `extra_well_known_type_scripts` and `extra_well_known_lock_scripts`

* Tell users which script is errored(most developers of DApps will encounter this error at least once.)

Reply:
I plan to modify this wiki page after this PR and [cheque] rfc PR (https://github.com/nervosnetwork/rfcs/pull/241) are merged.

Reply:
> * Tell users which script is errored(most developers of DApps will encounter this error at least once.)

RPC will report the error output index in line 475
```rust
.map_err(|err| format!("output index: {}, error: {:?}", index, err))
```

Reply:
An output has one lock-script and one type-script. Is it possible to distinguish between lock-script and type-script errors?

Reply:
resolved in commit https://github.com/nervosnetwork/ckb/pull/2821/commits/deedc21f03c0e0737e0dc754df8a95ce351b7d91